src/errors/test-run/index.js

```diff
@@ -298,11 +300,12 @@ export class ActionUnsupportedDeviceTypeError extends ActionArgumentErrorBase {
 
 // Selector errors
 export class ActionSelectorError extends TestRunErrorBase {
-    constructor (selectorName, errMsg) {
+    constructor (selectorName, err, isAPIError) {
```

Comment:
I think we can avoid passing `isAPIError` through the constructor argument and just use `instanceof`, can't we?

Reply:
unfortunately not, we can't import `APIError` here because of server side dependencies

Reply:
Aww, snap, you're right.